varanny.go

```diff
@@ -145,6 +148,17 @@ func getConfig(path string) (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
+
```

Comment:
I refactored where defaults for unset values in the config file are interpreted. They're now all here, just after the JSON is parsed